sonar-project.properties

```diff
@@ -6,4 +6,4 @@ sonar.javascript.lcov.reportPaths=coverage/lcov.info
 # Only scan code under src.
 sonar.sources=src
 # Skip test files, which can be anywhere in the source tree.
-sonar.exclusions=**/**.test.js
+sonar.exclusions=**/**.test.js,**/**.test.ts
```

Comment:
Interesting, so if it's excluded from the scan completely, I guess it doesn't need to be separately excluded from the coverage setting.

Reply:
This is based on how we currently handle `.test.js` files (which don't show up in the coverage report) and https://docs.sonarcloud.io/advanced-setup/analysis-scope/, which says:

> `sonar.sources` defines the initial scope of analysis for non-test code

and 

> Why is test code scoped separately?
>
> Test and non-test code are distinguished because different analysis rules are applied to each category; the two > categories have different metrics.
>
> Additionally, test code does not counts toward your lines of code (LOC) usage in paid SonarCloud accounts and does not count toward coverage (you don’t have to test your test code).